sdk/keyvault/azure-keyvault-keys/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 4.2.0b1 (2020-03-10)
+- Support for Key Vault API version 7.1-preview
+([#10124](https://github.com/Azure/azure-sdk-for-python/pull/10124))
+  - Added `import_key` to `KeyOperation`
```

Comment:
The value should just be "import". Do you add "_key" after every enum value?

Reply:
No, just this one: `import` is a keyword. I liked `import_key` better than autorest's `import_enum`.